x-pack/legacy/plugins/siem/public/components/page/hosts/events_table/index.tsx

```diff
@@ -16,33 +16,53 @@ import { Ecs } from '../../../../graphql/types';
 import { hostsModel, hostsSelectors, State } from '../../../../store';
 import { getEmptyTagValue, getOrEmptyTag } from '../../../empty_value';
 import { HostDetailsLink, IPDetailsLink } from '../../../links';
-import { Columns, ItemsPerRow, LoadMoreTable } from '../../../load_more_table';
+import { Columns, ItemsPerRow, PaginatedTable } from '../../../paginated_table';
 import { getRowItemDraggable, getRowItemDraggables, OverflowField } from '../../../tables/helpers';
 import { PreferenceFormattedDate } from '../../../formatted_date';
 import { LocalizedDateTooltip } from '../../../localized_date_tooltip';
 
 import * as i18n from './translations';
-
+const tableType = hostsModel.HostsTableType.events;
 interface OwnProps {
   data: Ecs[];
-  loading: boolean;
-  hasNextPage: boolean;
+  fakeTotalCount: number;
   id: string;
-  nextCursor: string;
-  tiebreaker: string;
+  loading: boolean;
+  loadPage: (newActivePage: number) => void;
+  showMorePagesIndicator: boolean;
   totalCount: number;
-  loadMore: (cursor: string, tiebreaker: string) => void;
   type: hostsModel.HostsType;
 }
 
 interface EventsTableReduxProps {
+  activePage: number;
   limit: number;
 }
 
 interface EventsTableDispatchProps {
-  updateLimitPagination: ActionCreator<{ limit: number; hostsType: hostsModel.HostsType }>;
+  updateTableActivePage: ActionCreator<{
+    activePage: number;
+    hostsType: hostsModel.HostsType;
+    tableType: hostsModel.HostsTableType;
+  }>;
+  updateTableLimit: ActionCreator<{
+    limit: number;
+    hostsType: hostsModel.HostsType;
+    tableType: hostsModel.HostsTableType;
+  }>;
 }
 
+export declare type EventsTableColumns = [
```

Comment:
I see the `declare` keyword being used in a few places now:

<img width="583" alt="Screen Shot 2019-07-19 at 12 32 18 PM" src="https://user-images.githubusercontent.com/1151048/61557368-4af93700-aa21-11e9-9336-faa013fbf44d.png">

But isn't that primarily only used for ambient type definitions and files?
https://stackoverflow.com/questions/35019987/what-does-declare-do-in-export-declare-class-actions

I took it out of all the areas and ran the type checker and everything seems to pass the type checker indicating that it's not really doing anything in these cases.

My recommendation would be to remove 'em.